admin/class-admin-apple-news.php

```diff
@@ -79,6 +83,24 @@ public function __construct() {
 
 		// Add JSON customization support.
 		new Admin_Apple_JSON();
+
+		if( function_exists( 'is_gutenberg_page' ) ) { // check if GB is active
```

Comment:
We need a generic function for determining if Gutenberg is active for a post, and we need to use it here.

In `includes/class-apple-news.php` I am doing this:

```php
if ( get_the_ID() && use_block_editor_for_post( get_the_ID() ) ) {
```

We need to split that out into a util function, which can live in the main Apple News file in the root, something like:


```php
function apple_news_block_editor_is_active( $post_id = 0 ) {
    if ( empty( $post_id ) ) {
        $post_id = get_the_ID();
    }
    return $post_id && use_block_editor_for_post( $post_id );
}
```

Reply:
```suggestion
		if ( function_exists( 'is_gutenberg_page' ) ) { // check if GB is active
```